astronomer/providers/amazon/aws/sensors/s3.py

```diff
@@ -200,39 +201,16 @@ class S3KeysUnchangedSensorAsync(BaseOperator):
         when this happens. If false an error will be raised.
     """
 
-    template_fields: Sequence[str] = ("bucket_name", "prefix")
-
     def __init__(
         self,
-        *,
-        bucket_name: str,
-        prefix: str,
-        aws_conn_id: str = "aws_default",
-        verify: Optional[Union[bool, str]] = None,
-        inactivity_period: float = 60 * 60,
-        min_objects: int = 1,
-        previous_objects: Optional[Set[str]] = None,
-        allow_delete: bool = True,
         **kwargs: Any,
     ) -> None:
         super().__init__(**kwargs)
-        self.bucket_name = bucket_name
```

Comment:
why are this params are been removed ?

Reply:
We were using BaseOperator before so we had added this now it's from `S3KeysUnchangedSensor` so due to inheritance these variables are already present.